api/src/paths/permit-no-sampling.test.ts

```diff
@@ -0,0 +1,135 @@
+import chai, { expect } from 'chai';
+import { describe } from 'mocha';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import * as permit_no_sampling from './permit-no-sampling';
+import * as db from '../database/db';
+
+chai.use(sinonChai);
+
+describe('permit-no-sampling', () => {
+  const dbConnectionObj = {
```

Comment:
No necessarily in this PR, but we should probably move some of these base common mocks into a folder somewhere, so we dont have to re-create these common mocks everywhere. Similar to the test-helpers in the frontend.  And then add thtat folder to the sonar ignore, etc.

Reply:
for sure will do it in a later pr when touching api tests next